samples/02_commands_framework/Modules/PublicModule.cs

```diff
@@ -59,5 +85,18 @@ public Task EchoAsync([Remainder] string text)
         [Command("list")]
         public Task ListAsync(params string[] objects)
             => ReplyAsync("You listed: " + string.Join("; ", objects));
+
+        // Can be used to example put information
+        [Command("info")]
+        public async Task InformationAsync()
```

Comment:
IMO this command is rather pointless

Reply:
Right, this is pointless command - only i wanted show how to use embedBuilder; now i created embedBuilder in kick command.